src/components/UriViewer.js

```diff
@@ -183,6 +187,25 @@ const UriViewer = _.flow(
         })
         const metadata = await loadObject(googleProject, bucket, name)
         setMetadata(metadata)
+      } else if (isAzureUri(uri)) {
+        const provider = AzureBlobStorageFileBrowserProvider(
+          { workspaceId: workspace.workspace.workspaceId, pageSize: 1 }
+        )
+        const getFile = await provider.getFilesInDirectory('').then(files => {
```

Comment:
I'm confused here... It looks like this is always getting metadata for the first file at the top level of the storage container, not the file referenced by `uri`?